node/src/components/rpc_server/rpcs/state.rs

```diff
@@ -214,7 +214,7 @@ impl RpcWithParamsExt for GetItem {
             let result = Self::ResponseResult {
                 api_version,
                 stored_value,
-                merkle_proof: hex::encode(proof_bytes),
+                merkle_proof: base16::encode_lower(&proof_bytes),
```

Comment:
Note that the client was unnecessarily running the checksum algorithm.

It wouldn't matter to use use the checksum hex function since these are always going to be pretty long, but it's fine to encode them this way.

Reply:
Noted.  Fixed in [e93dc37](https://github.com/nilz3ro/casper-node/pull/5/commits/e93dc3787a9983c352ebfdba0fc6bccb8f4d7c41).